### DISCUSSION TITLE
OCP\Files\NotFoundException: /Lydia/files/user.nextcloud_export

### DISCUSSION THREAD

**Discussion by SophyInBetween** [nextcloud/all-in-one#3034] [Questions] [answered]
I got the following error after a reset of an AIO Instance running on virtual server ubuntu 22

[no app in context] Error: OCP\Files\NotFoundException: /Lydia/files/user.nextcloud_export at <<closure>>

 0. /var/www/html/lib/private/Files/Node/LazyUserFolder.php line 59
    OC\Files\Node\Root->get("/Lydia/files/user.nextcloud_export")
 1. /var/www/html/custom_apps/user_migration/lib/Notification/Notifier.php line 232
    OC\Files\Node\LazyUserFolder->get("user.nextcloud_export")
 2. /var/www/html/custom_apps/user_migration/lib/Notification/Notifier.php line 121
    OCA\UserMigration\Notification\Notifier->getExportFile(["OC\\User\\User"])
 3. /var/www/html/custom_apps/user_migration/lib/Notification/Notifier.php line 74
    OCA\UserMigration\Notification\Notifier->handleExportDone(["OC\\Notification\\Notification"], "en")
 4. /var/www/html/lib/private/Notification/Manager.php line 368
    OCA\UserMigration\Notification\Notifier->prepare(["OC\\Notification\\Notification"], "en")
 5. /var/www/html/apps/notifications/lib/Controller/EndpointController.php line 108
    OC\Notification\Manager->prepare(["OC\\Notification\\Notification"], "en")
 6. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 230
    OCA\Notifications\Controller\EndpointController->listNotifications("v2")
 7. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 137
    OC\AppFramework\Http\Dispatcher->executeController(["OCA\\Notificat ... "], "listNotifications")
 8. /var/www/html/lib/private/AppFramework/App.php line 183
    OC\AppFramework\Http\Dispatcher->dispatch(["OCA\\Notificat ... "], "listNotifications")
 9. /var/www/html/lib/private/Route/Router.php line 315
    OC\AppFramework\App::main("OCA\\Notificati ... r", "listNotifications", ["OC\\AppFramewo ... "], ["v2","ocs.notif ... "])
10. /var/www/html/ocs/v1.php line 64
    OC\Route\Router->match("/ocsapp/apps/no ... s")
11. /var/www/html/ocs/v2.php line 23
    require_once("/var/www/html/ocs/v1.php")

GET /ocs/v2.php/apps/notifications/api/v2/notifications
from 95.90.202.39 by Lydia at 2023-07-26T08:17:24+00:00

**Comment by szaimen** [comment, marked as answer]
Duplicate of https://github.com/nextcloud/user_migration/issues/287
